lib/Dialect/FIRRTL/FIRRTLOps.cpp

```diff
@@ -831,6 +857,28 @@ Operation *InstanceOp::getReferencedModule() {
   return circuit.lookupSymbol(moduleName());
 }
 
+void InstanceOp::build(OpBuilder &builder, OperationState &result,
+                       TypeRange resultTypes, StringRef moduleName,
+                       StringRef name, ArrayRef<Attribute> annotations,
+                       ArrayRef<Attribute> portAnnotations) {
+  result.addAttribute("moduleName", builder.getSymbolRefAttr(moduleName));
+  result.addAttribute("name", builder.getStringAttr(name));
+  result.addAttribute("annotations", builder.getArrayAttr(annotations));
+  result.addTypes(resultTypes);
+
+  if (portAnnotations.empty()) {
+    SmallVector<Attribute, 16> portAnnotationsVec;
+    for (unsigned i = 0, e = resultTypes.size(); i < e; ++i)
+      portAnnotationsVec.push_back(builder.getArrayAttr({}));
```

Comment:
This can be simplified a bit to:

`SmallVector<Attribute, 16> portAnnotationsVec(resultTypes.size(), builder.getArrayAttr({}));`